### ISSUE TITLE
🚀 - Ionic input/textarea can't be modified when ion-popover is open

### ISSUE THREAD

**Issue by saschwarz** [hakimio/ngx-ui-tour#184]
### Which ngx-ui-tour-* package is relevant to the feature request?

ionic

### Description

My tour walks users through entering data so I have disabled the backdrop. Unfortunately ion-popover captures the keyboard focus by default. So mouse clicks are allowed but not keyboard input.

That functionality can be disabled by setting `focusTrap=false` attribute on the ion-popover 
https://ionicframework.com/docs/api/popover#focustrap

It would be helpful to expose this flag in the ngx-ui-tour API.

**Comment by hakimio** [comment]
The config was just added in [v8.1.0](https://github.com/ionic-team/ionic-framework/releases/tag/v8.1.0).
I'll try to release a new major version of `ngx-ui-tour-ionic` which uses this config and drops support for `ionic v7`.

**Comment by saschwarz** [comment]
If anyone needs a workaround:

```
      this.tourService.stepShow$.subscribe(({ step }) => {
        // you could conditionally add/remove the class - here I slam it on every step which is unnecessary
        const popover = document.querySelector('ion-popover.tour-popover');
        if (popover) {
          // add custom class to ion-popover.tour-popover element to allow users to type in input fields
          // https://github.com/ionic-team/ionic-framework/pull/29379/files
          popover.classList.add('ion-disable-focus-trap');
        }
      });
```

**Comment by hakimio** [comment]
Unfortunately Angular `ion-popover` seems to be missing `focusTrap` input.
I have crate a new issue on their end: https://github.com/ionic-team/ionic-framework/issues/29728

**Comment by hakimio** [comment]
@saschwarz I have released [v5.0.0](https://github.com/hakimio/ngx-ui-tour/releases/tag/ionic%40v5.0.0) with `trapFocus` config.
